src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -23,11 +52,46 @@ public String getCompanyNumber() {
         return companyNumber;
     }
 
+    public Map<String, String> getAttachments() {
+        return attachments;
+    }
+
+    public String getReferenceNumber() {
+        return referenceNumber;
+    }
+
+    public String getCustomerEmail() {
+        return customerEmail;
+    }
+
+    public String getReason() {
+        return reason;
+    }
+
+    private Map<String, String> buildAttachmentsMap(String downloadPrefix,
+                                                    List<Attachment> attachments) {
+        Map<String, String> attachmentsMap = new HashMap<>();
+        for (Attachment attachment : attachments) {
+            String name = attachment.getName();
+            Links links = attachment.getLinks();
+            if(links != null) {
+                String downloadLink = String.format("%s%s", downloadPrefix,
+                        links.getLink(CoreLinkKeys.SELF));
+                attachmentsMap.put(name, downloadLink);
+            }
+        }
+        return attachmentsMap;
+    }
+
     @Override
     public String toString() {
         return "ChipsRequest{" +
             "objectionId='" + objectionId + '\'' +
             ",companyNumber='" + companyNumber + '\'' +
+            ",attachments='" + attachments.values() + '\'' +
```

Comment:
attachment names are missing from the toString, it only shows the values (urls)

Reply:
Had to build a string to get keys as well.